src/test/java/com/ss/greencity/util/WaitsSwitcher.java

```diff
@@ -0,0 +1,91 @@
+package com.ss.greencity.util;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+
+/**
+ * A class intended to help easily switching between implicit and explicit waits
+ */
+public final class WaitsSwitcher {
```

Comment:
An instance of this class is inserted into TestRunner, so just call waitsSwitcher in your tests